src/App.jsx

```diff
@@ -1,35 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// import React from "react";
+import "./App.css";
+import WrapTheList from "./WrapTheList";
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  {
+    id: 0,
+    title: 'Read 37 pages of "The Road To React 2023',
+  },
+  {
+    id: 1,
+    title: "Watch videos from week 1",
+  },
+  {
+    id: 2,
+    title: "Complete week 1 assignment",
+  },
+];
+
+/* // This is the initial implementation of the function we were expected to write for the assessment.
+const wrapTheList = (list) => {
+  return list.map((listItem) => {
+    return (
+      <li key={listItem.id}> {listItem.title} </li>
+    );
+  });
+}; */
```

Comment:
Depending on future lesson content, it might be important to use this implementation in the future, so I'm glad to see you've kept it here in a commented form.